apps/nuxt-ssr/nuxt.config.js

```diff
@@ -5,22 +5,22 @@ export default {
   head: {
     title: "ssr-catalogue",
     htmlAttrs: {
-      lang: "en"
+      lang: "en",
     },
     meta: [
       { charset: "utf-8" },
       { name: "viewport", content: "width=device-width, initial-scale=1" },
       { hid: "description", name: "description", content: "" },
-      { name: "format-detection", content: "telephone=no" }
+      { name: "format-detection", content: "telephone=no" },
     ],
     link: [
       { rel: "icon", type: "image/x-icon", href: "/favicon.ico" },
-      { rel: "stylesheet", href: "theme.css" }
-    ]
+      { rel: "stylesheet", href: "theme.css" },
+    ],
   },
 
   // Global CSS: https://go.nuxtjs.dev/config-css
-  css: [],
+  css: ["assets/css/style.css"],
```

Comment:
how does this interfer with themes?

Reply:
these are the components styles, like all css if global css is used these would interfer / interact with other ccs ( the c in  css). Most / all components use scoped styles however ( the vue compiler adds data tags to the html to make sure the css only effects the 'scoped' components and does not interfere with other ( theme) css. The effects are the same as the non-ssr styles